lib/screens/movie_details.dart

```diff
@@ -0,0 +1,123 @@
+import 'package:flutter/material.dart';
+
+import '../constants.dart';
+import '../custom_widgets.dart';
+import '../models/movie.dart';
+import '../repositories/genre_repository.dart';
+
+class MovieDetails extends StatefulWidget {
+  const MovieDetails({
+    super.key,
+    required this.movie,
+  });
+
+  final Movie movie;
+
+  @override
+  State<MovieDetails> createState() => _MovieDetailsState();
+}
+
+class _MovieDetailsState extends State<MovieDetails> {
+  late int _counter = widget.movie.voteCount;
+  static const double subHeaderPosition = 160.0;
+  static const String originalTitle = 'Original title:';
+  static const String overviewTitle = 'Overview';
+  static const String genreTitle = "Genres";
+
+  @override
+  Widget build(BuildContext context) {
+    void incrementLikeCounter() {
+      setState(() {
+        _counter++;
+      });
+    }
```

Comment:
let's move this outside the build method

Reply:
done